types/priv_validator.go

```diff
@@ -218,17 +219,16 @@ func (privVal *PrivValidatorFS) Save() {
 
 func (privVal *PrivValidatorFS) save() {
 	if privVal.filePath == "" {
-		cmn.PanicSanity("Cannot save PrivValidator: filePath not set")
+		panic("Cannot save PrivValidator: filePath not set")
 	}
 	jsonBytes, err := json.Marshal(privVal)
 	if err != nil {
-		// `@; BOOM!!!
-		cmn.PanicCrisis(err)
+		panic(err)
 	}
-	err = cmn.WriteFileAtomic(privVal.filePath, jsonBytes, 0600)
+	outFile := privVal.filePath
```

Comment:
would be good to move this up (replace `if privVal.filePath == ""` with `outFile == ""`)